### DISCUSSION TITLE
Using edge attributes with HGTConv

### DISCUSSION THREAD

**Discussion by huangd999** [pyg-team/pytorch_geometric#8302] [Q&A]
I am working on an edge classification problem on a heterogenous graph. Currently we are using HGTConv as part of our model. Our data includes edge attributes that we'd like to input into the model also. Is there a way to do this with HGTConv? I am under the impression that it only takes node attributes.

**Comment by rusty1s** [comment]
Currently, `HGTConv` cannot take edge attributes as input. Edge feature support would be supported by `to_hetero` or `HeteroConv`.
